server/src/main/java/com/lunchtime/LunchTimeApplication.java

```diff
@@ -35,9 +27,12 @@ public void addCorsMappings(CorsRegistry registry) {
                     // this list without creating new PR, for example for prod env, you will need only ssh
                     .allowedOrigins("http://localhost:3000");
             }
+
+            @Override
+            public void addResourceHandlers(ResourceHandlerRegistry registry) {
+                registry.addResourceHandler("/api/**")
+                    .addResourceLocations("classpath:/static/");
+            }
```

Comment:
It is Ok, but as i know (and see on **[github](https://github.com/search?p=1&q=addResourceHandlers%28ResourceHandlerRegistry+registry%29&type=Code)**), this should be in configuration class, but not in the main launch one.

Reply:
Fixed